### DISCUSSION TITLE
Github pages not issuing SSL certificate

### DISCUSSION THREAD

**Discussion by edazpotato** [community/community#23666] [Other Feature Feedback, Questions, & Ideas]
<p>I have a github pages site that is being published at custom domain (edaz.codes), and it has been working fine up till now but a few days ago it started issuing a new certificate and it has been showing this for the last few days. What do I need to do to fix this?<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/d24d105a8540c1c246cff3da49204367f72e6035.png" data-download-href="/uploads/short-url/u0pnoIB8AsGHujOsUktXSirwIxn.png?dl=1" title="" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/46797041/181078685-d6ffb520-5a56-4380-8104-0c76f1857e30.png" alt="" data-base62-sha1="u0pnoIB8AsGHujOsUktXSirwIxn" width="690" height="95" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d24d105a8540c1c246cff3da49204367f72e6035_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename"></span><span class="informations">919×127 3.67 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/edazpotato">@edazpotato</a>, welcome to the GitHub Support Community!</p>
<p>It looks like you’ve got the same problem as this user had in this thread—your DNS is set up with Cloudflare and GitHub isn’t able to see the required DNS entries to generate your SSL certificate. You’ll need to follow the steps in here to disable Cloudflare’s DNS proxy in order to let GitHub generate your certificate:</p>
<aside class="quote quote-modified" data-post="2" data-topic="134467">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/why-does-my-github-pages-service-always-show-that-ssl-certificates-are-not-allocated/134467/2">Why does my Github Pages service always show that SSL certificates are not allocated?</a> <a class="badge-wrapper  bullet" href="/c/github-help/github-pages/31"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #c8e1ff;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Discussion and questions about GitHub Pages—websites for you and your projects hosted right on your GitHub repository.">GitHub Pages</span></a>
  </div>
  <blockquote>
    Hi <a class="mention" href="/xiezheyuan">@XieZheyuan</a>, welcome to the GitHub Support Community! 
Cloudflare enables their HTTP/DNS proxy feature by default, meaning GitHub isn’t able to see the DNS records required to generate an HTTPS certificate. You’ll need to disable this for any DNS records that point towards GitHub. 
Disclaimer : Disabling the HTTP/DNS proxy may interfere with other Cloudflare services. If you use other Cloudflare services with your site, we aren’t able to guarantee that these will continue working. For more he…
  </blockquote>
</aside>

**Comment by kaanchan** [comment]
I have the same issue but don't even see a progress bar. My domain is verified in the Pages page. It has been several hours and I've removed and reverfied the domain twice now. My DNS is entries are hosted by NameSilo. A Claude scan of the entries said it didn't see anything that would block the generation. How can I troubleshoot further.
